embedded-graphics/src/primitives/circle.rs

```diff
@@ -233,6 +233,51 @@ mod tests {
     use super::*;
     use crate::pixelcolor::BinaryColor;
 
+    fn test_expected_circle(
```

Comment:
Couldn't you use ` MockDisplay::from_pattern` for these tests?

Reply:
Oop yeah I can. I just saw the `Line` harness and copied that. Tests now ported to `MockDisplay`. I had to add `#[rustfmt::skip]` to get the expected values to break lines. See https://github.com/jamwaffles/embedded-graphics/pull/209/files#r351842083

Reply:
I couldn't find a rule in the [config options](https://rust-lang.github.io/rustfmt/) to make it more specific so I've just disabled all formatting for that block with `#[rustfmt::skip]`.